packages/appeals-service-api/src/routes/v2/appeal-cases/spec.yaml

```diff
@@ -54,7 +54,113 @@ paths:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/AppealCase'
+              type: object
+              required:
```

Comment:
openApiValidatorMiddleware doesn't seem to actually require these, can't tell why if anyone can see why
could also do with automatically adding the schema's from data model into the api spec for these puts from BO topics